app/graphql/types/query_type.rb

```diff
@@ -1,17 +1,24 @@
 module Types
   class QueryType < Types::BaseObject
-    # Add root-level fields here.
-    # They will be entry points for queries on your schema.
 
-    # First describe the field signature:
+    field :all_media_entries, [Types::MediaEntryType], null: true do
+      description 'Find all MediaEntries'
+      argument :first, Int, required: false
+      argument :orderBy, MadekGraphqlSchema::OrderByEnum, required: false
+    end
+
     field :media_entry, MediaEntryType, null: true do
-      description "Find a MediaEntry by ID"
+      description 'Find a MediaEntry by ID'
       argument :id, ID, required: true
     end
 
-    # Then provide an implementation:
     def media_entry(id:)
       MediaEntry.find(id)
     end
+
+    def all_media_entries(first: 100, order_by: 'created_at DESC')
```

Comment:
The name of the method says something quite different it returns. Please consider a more appropriate one.

Reply:
@michalpodlecki I think the method name is appropriate because it reflects the name of the field in the API (that I requested).
Note that `allFoos` is just 1 of several conventions to name the (in REST-terms) "index".

Some examples, looking at it next to each other I think I prefer the last option… but the codebase is still small and it could be renamed later.

- `allFoos`: https://www.howtographql.com/basics/2-core-concepts/
- `fooConnection`: https://github.com/Shopify/graphql-design-tutorial/blob/master/TUTORIAL.md#lists-and-pagination
- `foos`: https://developer.github.com/v4/guides/migrating-from-rest/#example-nesting

Reply:
@eins78 what I meant - `all` in method name and `first` in attributes list are mutually exclusive ;) I'd remove the prefix from the method name and everything will be less confusing.

Reply:
@eins78 I read the docs and if above follows conventions, then it's OK :) It looked strange to me when I saw it the first time, so I instantly reacted :)